scaaml/capture/scope/scope_template.py

```diff
@@ -39,7 +46,7 @@ def arm(self):
         """Setup scope to begin capture when triggered."""
 
     @abstractmethod
-    def capture(self):
+    def capture(self, poll_done: bool = False) -> bool:
```

Comment:
Same thing here regarding prototype. I don't see what `poll_done` would bring. Removing it from the function signature and putting the default value in `Chipwhisperer.capture()` should do the trick for us. It's not a value we change.
If later we need extra arguments to support a specific scope, we can revisit it by adding `**kwargs` to the function.

Please document return value.

Reply:
Same error with incompatible definition. `ChipwhispererScope` inherits both `OpenADC` and `ScopeTemplate` thus the base classes must have same signatures. Am I not understanding something?